portal/models.py

```diff
@@ -112,6 +112,16 @@ def teaches(self, userprofile):
             student = userprofile.student
             return not student.is_independent() and student.class_field.teacher == self
 
+    def has_school(self):
+        return self.school is not (None or "")
+
+    def has_class(self):
+        classes = self.class_teacher.all()
+        if classes.count() == 0:
+            return False
+        else:
+            return True
```

Comment:
Since classes.count() == 0 is a boolean result, we can return this... or specifically
return classes.count() != 0